### PR TITLE
Fix: Fixed inconsistencies, issues caused by sbox updates and mouse detection

### PR DESCRIPTION
**Pull request by Alf21**

### CODE REVIEW

File: code/ui/Hud.cs [hunk 1]
```diff
@@ -86,16 +88,51 @@ public GeneralHud()
             }
         }
 
-        public class AliveHud : Panel
+        public class AliveHud
         {
-            public AliveHud()
+            public bool Enabled
```
Comment (mzegar) [inline code/ui/Hud.cs:93]: Could technically inherit from panel and then override the `Enabled` behaviour (I'm pretty sure)

Comment (Alf21) [reply code/ui/Hud.cs:93]: We don't need multiple panel instances, that would be incompatible anyways as there gonna be issues with z-index and ordering. With this change, you can mix panels with general hud panels, theoretically.

**Review by mzegar — APPROVED** [review]
Up to you on the one comment, you would need to define `_enabled`, but you'd still have to define the same overridden behaviour

File: code/ui/Hud.cs (hunk 1 above)
Comment (mzegar) [reply code/ui/Hud.cs:93]: yeah nvm, there isn't any reason for this.

**Comment by GravityWolfNotAmused** [comment]
Awaiting: #229 to handle possible merge conflicts with this PR.

**Pull request merged**